node-module-system/calculate.js

```diff
@@ -0,0 +1,22 @@
+import add from './add.js';
+import subtract from './subtract.js';
+import multiply from './multiply.js';
+import divide from './divide.js';
+
+const num1 = process.argv[2];
+const num2 = process.argv[4];
+const operation = process.argv[3];
+
+let result;
+
+if (operation === 'plus') {
+  result = add(Number(num1), Number(num2));
```

Comment:
Wouldn't it be easier to do the Number conversions up where you extract the arguments?